app/assets/javascripts/govuk_publishing_components/vendor/polyfills/common.js

```diff
@@ -0,0 +1,8 @@
+function nodeListForEach (nodes, callback) {
```

Comment:
If this is still required can this commit be squashed into the commit that removed it?

Where is it used? When I saw you'd removed it I had a look to confirm and couldn't see any other places where it is called.